src/applications/caregivers/tests/unit/components/ApplicationDownloadLink.unit.spec.js

```diff
@@ -79,21 +107,10 @@ describe('CG <ApplicationDownloadLink>', () => {
       revokeObjectStub.restore();
     });
 
-    it('displays error message and records error when request fails', async () => {
-      const { getByText, container } = subject();
-      const link = $('va-link', container);
-      expect(link).to.exist;
-      expect(container.querySelector('va-loading-indicator')).to.not.exist;
-
-      mockApiRequest({}, false);
+    it('should record error event when the request fails', async () => {
```

Comment:
Split the error message and event recording into separate tests to accommodate for the branching in the message logic.